pkgs/development/python-modules/pypemicro/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, buildPythonPackage, fetchPypi, autoPatchelfHook }:
+
+buildPythonPackage rec {
+  pname = "pypemicro";
+  version = "0.1.9";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-HouDBlqfokKhbdWWDCfaUJrqIEC5f+sSnVmsrRseFmU=";
+  };
+
+  pythonImportsCheck = [ "pypemicro" ];
+
```

Comment:
Are we executing the tests https://github.com/NXPmicro/pypemicro/tree/master/tests ?

Reply:
Then please add ``doCheck = false`` with a short comment